libflux/flux-core/src/formatter/tests.rs

```diff
@@ -155,6 +155,7 @@ fn dict() {
     "c": 2,
 ]"#,
     );
+    assert_unchanged(r#"[: ]"#);
```

Comment:
I'd prefer the formatted source to be `[:]` without the space in the middle. Thoughts?

Reply:
Yes, that can be done. I followed the existing ': space' format, but it's written that way since there's the right element after it, but there's no right element in our case. I will change it.